dash_dangerously_set_inner_html/package-info.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
In the past I've avoided contributing the `package-info.json`, adding just the changes to `package.json` instead.